SemiTruck.h

```diff
@@ -1,2 +1,16 @@
 #pragma once
+#include "Vehicle.h"
+#include <string>
 
+struct SemiTruck : Vehicle
+{
+    SemiTruck(const std::string& s);
+
+    ~SemiTruck() override;
+    SemiTruck(const SemiTruck&);
+    SemiTruck& operator=(const SemiTruck&)= default;
```

Comment:
Move this implementation to the cpp file
Add space around your `=` to improve readability
- [ ] revise